src/DynamoCoreWpf/ViewModels/Core/Converters/DynamoTextOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Wpf.ViewModels.Core.Converters
+{
+    public class DynamoTextOptions
+    {
+        private static ICSharpCode.AvalonEdit.TextEditorOptions TextOptions;
+
+        /// <summary>
+        /// Class to interact with ICSharp Text Editor Options
+        /// </summary>
+        public DynamoTextOptions()
+        {
+            if(TextOptions == null)
+                TextOptions = new ICSharpCode.AvalonEdit.TextEditorOptions();
+        }
+        /// <summary>
+        /// Sets the text editor options
+        /// </summary>
+        public void SetTextOptions(bool value) {
+            TextOptions.ShowSpaces = value;
+            TextOptions.ShowTabs = value;
+        }
+        /// <summary>
+        /// Gets the text editor options
+        /// </summary>
+        public ICSharpCode.AvalonEdit.TextEditorOptions GetTextOptions()
```

Comment:
If this ends up being `public` we'd still expose the type publicly in our API, right?

Can we maybe fire an event when the editor setting is changed and react to it on the Python side by changing a global `TextEditorOptions` that lives inside that assembly?

Reply:
Will try to do this.

Reply:
you can also maybe cheat and use internal and `internalsVisibleto`

Reply:
Cheating would be fine too. If you do please add a comment stating it was made internal to avoid exposing AvalonEdit in our API.